ansible/group_vars/wakuv2-test.yml

```diff
@@ -29,6 +29,8 @@ nim_waku_rpc_tcp_port: 8545
 
 # Limits
 nim_waku_p2p_max_connections: 300
+nim_waku_ip_colocation_limit: 2
+nim waku_max_msg_size: 'size:150KiB'
```

Comment:
The flags don't exist in the role used to deploy nim-waku: 
* https://github.com/status-im/infra-role-nim-waku/blob/master/templates/docker-compose.yml.j2.
You can find an example of PR to add flags : https://github.com/status-im/infra-role-nim-waku/pull/20/files

Feel free to contact me if you need help

Reply:
@apentori https://github.com/status-im/infra-role-nim-waku/pull/24